Resources/Textures/_RMC14/Objects/Misc/redtennisball.rsi/meta.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA-3.0",
```

Comment:
This license isn't compatible with RMC unless the author provides us explicit permission.